tools/coredump/modulestore/store.go

```diff
@@ -332,6 +333,36 @@ func (store *Store) UnpackModule(id ID, out io.Writer) error {
 	return nil
 }
 
+// ExportModule exports a compressed module from the store, writing it to the given writer.
+func (store *Store) ExportModule(id ID, tw *tar.Writer) error {
```

Comment:
I guess, this should not be part of this PR.

Reply:
Seems so, I merged `main` now and this should be gone.